components/list/list-item-drag-drop-mixin.js

```diff
@@ -335,6 +335,7 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 					opacity: 0;
 				}
 				:host([selected]) d2l-list-item-drag-handle,
+				d2l-list-item-drag-handle:hover,
```

Comment:
I almost wonder if the `pointer-events: none` change in [this PR](https://github.com/BrightspaceUI/core/pull/4100/files) is necessary with this change. I am a little worried about causing a regression on that fix if it's removed however.